projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -354,34 +402,78 @@ export class OwcJsonService {
   }
 
   /**
-   * Layer priority: a owc-context might offer a single layer in mulitple ways - as a wms, as a wmts, etc.
-   * We need to chose a single one of those offerings.
-   * The order we chose here is first wms, then wmts, then wfs, then others.
+   * layer priority: first wms, then wmts, then wfs, then others.
    */
-  public getLayers(owc: IOwsContext, targetProjection: string): Observable<Layer[]> {
+  public getLayers(owc: IOwsContext, targetProjection: string): Observable<(Layer | LayerGroup)[]> {
```

Comment:
Changing the return type is also a breaking Change.

Reply:
How should we proceed with these?

We've already decided during the sprint that some degree of breaking changes was acceptable this time - since our changes to the `dimension` field did constitute such a change, too.

I can revert this change, but returning layer-groups was one of the requested features, so I'd rather not do that.
I could create a separate method called `getLayersOrGroups` and leave `getLayers` like it is now, but those would be two methods doing almost the same thing - I think that would be confusing to the user.

Reply:
We can make such changes but we should try to avoid breaking Changes if possible :)

If we do them then mark the PR with:

> ## Does this PR introduce a breaking change?
> - [x] Yes
> - [ ]  No

and list these changes in `CHANGELOG.md` in a section

> ### Breaking Changes
> * **@dlr-eoc/package:** descript what is breaking